src/features/Licenses/data/thunks.js

```diff
@@ -8,11 +8,11 @@ import {
   fetchLicensesDataFailed,
 } from 'features/Licenses/data/slice';
 
-function fetchLicensesData(id, currentPage) {
+function fetchLicensesData(id, currentPage, urlParams) {
```

Comment:
Is better to name it filtersData because is the data you are passing in this parameter, the others variables(currentPage, institutionId) are also urlparams so is too general. Instead of changing the name variable, add the docs for the function describing that there are the url params to filter
```suggestion
function fetchLicensesData(id, currentPage, filtersData) {
```

Reply:
@01001110J you can name it same as the dispatch part, no problem with that